.gitignore

```diff
@@ -22,3 +22,4 @@
 .yardoc/
 /doc/
 /failures
+.DS_Store
```

Comment:
What Chris said. `.DS_Store` should be treated as an environment-specific file in the same vain as a `*.swp` Vim swapfile or a `sublime-project`.